### PR TITLE
fix(shared-data): extents update from hardware

### PR DESCRIPTION
**Pull request by ryanthecoder**
# Overview
Hardware got back to me with some updated numbers.

### CODE REVIEW

File: shared-data/pipette/definitions/2/general/ninety_six_channel/p200/3_0.json [hunk 1]
```diff
@@ -649,7 +649,7 @@
   "channels": 96,
   "shaftDiameter": 2,
   "shaftULperMM": 3.14159,
-  "backlashDistance": 3.0,
+  "backlashDistance": 0.3,
```
Comment (vegano1) [inline shared-data/pipette/definitions/2/general/ninety_six_channel/p200/3_0.json:652]: this feels like a big change, is this accurate?

**Review by vegano1 — APPROVED** [review]
left a comment

File: shared-data/pipette/definitions/2/general/ninety_six_channel/p200/3_0.json (hunk 1 above)
Comment (sfoster1) [reply shared-data/pipette/definitions/2/general/ninety_six_channel/p200/3_0.json:652]: yeah, that's the fundamental thing the hardware is improving

**Review by sfoster1 — APPROVED** [review]
nice, ty

File: shared-data/pipette/definitions/2/general/ninety_six_channel/p200/3_0.json (hunk 1 above)
Comment (ryanthecoder) [reply shared-data/pipette/definitions/2/general/ninety_six_channel/p200/3_0.json:652]: yea this was an accidental inclusion from a different pipette project, this is correct

**Pull request merged**